firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -473,4 +500,14 @@ private boolean isInstalledRelease(AppDistributionReleaseInternal latestRelease)
             ReleaseIdentificationUtils.extractInternalAppSharingArtifactId(
                 firebaseApp.getApplicationContext()));
   }
+
+  private void redirectToPlayForAabUpdate(String downloadUrl) {
+    Intent updateIntent = new Intent(Intent.ACTION_VIEW);
+    Uri uri = Uri.parse(downloadUrl);
```

Comment:
Since downloadUrl is nullable, we should probably add a check to make sure this string is properly formatted (or at least, not null) before redirecting to play